project_purchase_link/tests/test_project_purchase_link.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo.osv import expression
+from odoo.tests import common
+from odoo.tools.safe_eval import safe_eval
+
+
+class TestProjectPurchaseUtilities(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestProjectPurchaseUtilities, cls).setUpClass()
+        cls.project_model = cls.env["project.project"]
+        cls.project = cls.project_model.create({"name": "Test Project",})
+        cls.purchase_model = cls.env["purchase.order"]
+        cls.purchase = cls.purchase_model.search(
+            [("state", "in", ("draft", "sent")),], limit=1
```

Comment:
```suggestion
            [("state", "in", ("draft", "sent")), ], limit=1
```
or 
[("state", "in", ("draft", "sent"))], limit=1